s3/pkg/datastore/aws/aws.go

```diff
@@ -61,9 +65,49 @@ func (myc *s3Cred) IsExpired() bool {
 	return false
 }
 
+func (ad *AwsAdapter) BucketCreate(ctx context.Context, in *pb.Bucket) error {
+
+	log.Info("Bucket create is called in aws service")
```

Comment:
These two logs can be merged

Reply:
Both logs have different purposes. So kept separately